docs-website/static/llms.txt

```diff
@@ -6493,6 +6493,14 @@ If you’re a team running Haystack in production or want to move faster and sca
 
 👉 [Get in touch with our team to explore Haystack Enterprise](https://www.deepset.ai/products-and-services/haystack-enterprise)
 
```

Comment:
My impression is that we want this to be automatically generated every time. Agree?

If so, I'd propose removing this file from this PR and adding it to .gitignore